src/datasets/utils/resources/languages.json

```diff
@@ -523,6 +525,7 @@
     "gom": "Goan Konkani",
     "gor": "Gorontalo",
     "got": "Gothic",
+    "GR": "Greek",
```

Comment:
Than language code for Greek is `el`. I would remove `gr` from README and from here.
```suggestion
```